.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -20,6 +20,32 @@ This issue captures the state of the OpenSearch release, its assignee (Release M
 
 Please refer to the following link for the release version dates: [Release Schedule and Maintenance Policy](https://opensearch.org/releases.html).
 
+### [Entrance Criteria](https://github.com/opensearch-project/.github/blob/main/RELEASING.md#entrance-criteria-to-start-release-window)
+Criteria | Status | Description  | Comments
+-- | -- | -- | --
+Documentation draft PRs are up and in tech review for all component changes | :red_circle: |   |
+Sanity testing is done for all components | :red_circle: |   |
+Code coverage has not decreased (all new code has tests) | :red_circle: |   |
+Release notes are ready and available for all components | :red_circle: |   |
+Roadmap is up-to-date (information is available to create release highlights) | :red_circle: |   |
+Release ticket is cut, and there's a forum post announcing the start of the window | :red_circle: |   |
+[Any necessary security reviews](##Security-Reviews) are complete | :red_circle: |   |
+
+### OpenSearch {{ env.VERSION }} [exit criteria](https://github.com/opensearch-project/.github/blob/main/RELEASING.md#exit-criteria-to-close-release-window) status:
```

Comment:
@CEHENKLE Should we add "No unpatched vulnerabilities of medium or higher severity that have been publicly known for more than 60 days" to exit criteria as well?

Reply:
I put it into exit criteria, but yes :)